include/wx/desktopenv.h

```diff
@@ -0,0 +1,35 @@
+//////////////////////////////////////////////////////////////////////////////
+// Name:        wx/desktopenv.h
+// Purpose:     wxDesktopEnv class interface
+// Author:      Igor Korot
+// Created:     2019-10-07
+// Copyright:   (c) 2019 wxWidgets development team
+// Licence:     wxWidgets licence
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_DESKTOPENV_H_BASE_
+#define _WX_DESKTOPENV_H_BASE_
+
+#include "wx/defs.h"
+
+class WXDLLIMPEXP_CORE wxDesktopEnvBase
+{
+public:
+    wxDesktopEnvBase() { }
+    virtual ~wxDesktopEnvBase() { }
```

Comment:
It doesn't make sense to create objects of this class, it's a natural singleton, so just make it a namespace-like class, i.e. class with only static methods.